app/main.py

```diff
@@ -1,8 +1,47 @@
+from decimal import Decimal
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: int,
+                 average_rating: float, count_of_ratings: float) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list[Car]) -> float:
+        cost = 0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                cost += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return cost
+
+    def calculate_washing_price(self, car: Car) -> float:
+        if self.clean_power <= car.clean_mark:
+            return 0
+        price = round(float(
```

Comment:
There is no need to round or convert to float the expression in this method, perform rounding in the `serve_cars` method.

Reply:
Agree, there is such a requirement, then leave rounding.